internal/kafka/internal/workers/kafka_mgrs/kafkas_mgr.go

```diff
@@ -207,6 +207,11 @@ func (k *KafkaManager) calculateAvailableAndMaxCapacityForDynamicScaling(streami
 	// of Kafkas have already been created. In this case we limit the metric to 0 anyways.
 	available := math.Max(0, maxAvailable-float64(streamingUnitsByRegion.Count))
 
+	if streamingUnitsByRegion.ClusterType == api.EnterpriseDataPlaneClusterType.String() {
```

Comment:
Could you explain / expand why this is needed?

Reply:
I believe because region limits shouldn't be taken into consideration for enterprise clusters capacity. @machi1990 - can you confirm?

Reply:
+1 that's correct.

Regions limits are meant to be for managed clusters i.e our own clusters.
For the enterprise case, it could well be possible to have a cluster in a region that KFM does not have configuration for in its supported cloud provider configuration.

Reply:
Could we add a comment mentioning that?

Reply:
Also does that mean that the region limits do not apply to enterprise clusters? If so, can we document it too in the config?